docs/topics/plugin-icon-legend.adoc

```diff
@@ -5,17 +5,17 @@
 [id='plugin-icon-legend_{context}']
 = About {ProductShortName} issue icons
 
-In the Issue Explorer and file editors, {ProductShortName} issues use an icon to indicate their severity level and status. The following table describes the meaning of the various icons.
+{ProductShortName} displays icons in *Issue Explorer* and file editors to indicate the severity and status of issues related to migration. The table below describes the meaning of these icons.
```

Comment:
"... in the *Issue Explorer* and the file editors to indicate the severity and status of the issues." 
You can omit "related to migration".